app/src/main/java/com/asfoundation/wallet/subscriptions/SubscriptionDetailsPresenter.kt

```diff
@@ -35,12 +35,12 @@ class SubscriptionDetailsPresenter(
   }
 
   private fun onSubscriptionDetails(subscriptionDetails: SubscriptionDetails) {
-    view.showDetails()
     if (subscriptionDetails is ActiveSubscriptionDetails) {
       view.showActiveDetails(subscriptionDetails)
```

Comment:
Update the method name to `setActiveDetails` and `setExpiredDetails` since both method are only to set the content and the content is only shown when the `showDetails` is called

Reply:
[MOB-46] Updated screens after review feedback